queries/javascript/indents.scm

```diff
@@ -0,0 +1,25 @@
+; inherits: (jsx)
```

Comment:
jsx indents are not being imported in react js files. Works if you add indent and branch queries from jsx to javascript index.scm instead of inheritting. Might be due to some overriding

Reply:
hmm, it works on my end though. Let me double check it with a minimal init.vim

Reply:
Sure

Reply:
it does seem to work using this minimal config https://paste.sh/OIGOYy-6#0Djftco4hcAZ2dgfN2BA0kTI, don't forget to set the `indentexpr` to `nvim_treesitter#indent()`. Not sure why it has to be done it manually.

Also, the last `</>` for jsx fragment seems to be broken, it doesn't get outdented properly. I thought by adding `"</>"` to `@branch` would fix it but it didn't.

Reply:
setting indentexpr made it work haha